packages/flutter/lib/src/material/material.dart

```diff
@@ -89,18 +146,21 @@ class Material extends StatefulWidget {
   /// The widget below this widget in the tree.
   final Widget child;
 
+  /// The kind of material (e.g., card or canvas).
   final MaterialType type;
 
   /// The z-coordinate at which to place this material.
   final int elevation;
 
+  /// The color of the material.
```

Comment:
say if this can be transparent (probably not if you have an elevation, since you'd see the shadow below?)